### PR TITLE
Adding DSDS hint to eSIM section of usage guide.

### PR DESCRIPTION
**Pull request by jonathancross**
Adding info requested here: https://github.com/GrapheneOS/os-issue-tracker/issues/1196#issuecomment-1163011728

Also confirmed it is correct and solves the problem on pixel 4a.

### CODE REVIEW

File: static/usage.html [hunk 1]
```diff
@@ -999,6 +999,8 @@ <h3><a href="#sandboxed-google-play-esim">Privileged eSIM management</a></h3>
 
                     <p>By enabling the toggle, the proprietary Google functionality is enabled and
                     will be used by the OS to provision and manage eSIMs.</p>
+
+                    <p>NOTE: you may need to <a href="https://github.com/GrapheneOS/os-issue-tracker/issues/1196#issuecomment-1127466518">enable DSDS</a> if you find that you the eSIM installation process does not progress past "Checking network info..." stage.</p>
```
Comment (ghost) [inline static/usage.html]: 1. Avoid using all caps words unless they're technical terms for better readability. Use "Note you may [..]"
2. Please split all this up into individual line breaks so it's easier for editing the HTML. See how other paragraphs look in the source code for maximum line width.

Comment (ghost) [reply static/usage.html]: Also the code the user posted is not correct. It's `*#*#4636#*#*`

Comment (jonathancross) [reply static/usage.html]: Thanks.  I added the USSD code directly to this usage guide for clarity.

**Comment by flawedworld** [comment]
good to merge

**Pull request merged**